packages/aws-cdk-lib/aws-backup/lib/rule.ts

```diff
@@ -44,6 +44,17 @@ export interface BackupPlanRuleProps {
    */
   readonly scheduleExpression?: events.Schedule;
 
+  /**
+   * The name of the time zone for the schedule expression, following the IANA time zone standard.
+   *
+   * @example America/Los_Angeles
+   *
+   * @see https://www.iana.org/time-zones
+   *
+   * @default - UTC
+   */
+  readonly scheduleExpressionTimezone?: string;
```

Comment:
we export a `TimeZone` prop out of core.

Reply:
missed that one